packages/application-generic/src/services/bull-mq/old-instance-bull-mq.service.ts

```diff
@@ -49,7 +49,8 @@ export class OldInstanceBullMqService {
   constructor() {
     if (this.shouldInstantiate()) {
       this.inMemoryProviderService = new InMemoryProviderService(
-        InMemoryProviderEnum.OLD_INSTANCE_REDIS
+        InMemoryProviderEnum.OLD_INSTANCE_REDIS,
+        true
```

Comment:
We now have to pass the cluster mode as variable, therefore this setting here because the old Redis instance of the Workflow Engine is in cluster.